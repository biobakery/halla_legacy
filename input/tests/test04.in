TID-uniform_random_D1-3_spiked	C1	C2	C3	C4	C5	C6	C7	C8	C9	C10	C11	C12	C13	C14	C15	C16	C17	C18	C19	C20	C21	C22	C23	C24
M1	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	1	1	1	1	1	1	1	1
M2	0	0	0	0	0	0	0	0	1	1	1	1	1	1	1	1	2	2	2	2	2	2	2	2
M3	0.1	0.2	0.3	0.4	0.5	0.6	0.7	0.8	0.9	1	1.1	1.2	1.3	1.4	1.5	1.6	1.7	1.8	1.9	2	2.1	2.2	2.3	2.4
M4	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA
D1	0.117706508	0.125397695	0.185940646	0.009246573	0.099491099	0.12677216	0.130261419	0.129667066	0.017136141	0.070909518	0.151768835	0.125356444	0.3944516	0.35474963	0.307240418	0.335699898	0.340757139	0.351246112	0.236537966	0.276978742	0.277769043	0.249883704	0.292966472	0.32801971
D2	0.175972951	0.192723587	0.200963966	0.179713614	0.235931805	0.180585383	0.209396204	0.160492072	0.079457849	0.070057928	0.144086709	0.057135638	0.050612397	0.072439296	0.042408039	0.087525933	0.122541592	0.09014101	0.065128461	0.095271117	0.08756246	0.108894594	0.119138077	0.032881584
D3	0.036291004	0.053183196	0.058185175	0.111746174	0.115469832	0.104463023	0.132559803	0.112793779	0.169745859	0.248496975	0.199864079	0.178168048	0.226009446	0.240143797	0.194078445	0.226736884	0.277365588	0.262469704	0.201094425	0.214967738	0.282212373	0.270300529	0.2970539	0.282412114
D4	0.254230128	0.204695696	0.226021302	0.040734295	0.036640112	0.088322062	0.175987431	0.067864111	0.25615895	0.072112795	0.248967771	0.130691351	0.101504105	0.007796818	0.093102032	0.063022216	0.016447882	0.027513772	0.069481138	0.124271041	0.146926162	0.1262609	0.083012755	0.062884647
D5	0.07330171	0.155143846	0.023898456	0.168997004	0.113519167	0.256585307	0.129475888	0.157788482	0.148976425	0.182207523	0.043700102	0.132516311	0.060249725	0.078966459	0.162889146	0.141558547	0.017751839	0.118274505	0.151210659	0.12551167	0.010906538	0.053014736	0.087503688	0.035042794
D6	0.19055748	0.021961334	0.220510016	0.20465454	0.255626064	0.144862145	0.172017994	0.197677312	0.23385432	0.109811253	0.066500312	0.182101956	0.124966165	0.063344026	0.140825375	0.108598352	0.099921835	0.073577356	0.127881763	0.014007768	0.191754044	0.085088862	0.001220542	0.113708942
D7	0.151940219	0.246894646	0.08448044	0.284907801	0.143321921	0.098409921	0.05030126	0.17371718	0.094670457	0.246404008	0.145112192	0.194030252	0.042206564	0.182559974	0.059456547	0.036858169	0.125214124	0.07677754	0.148665589	0.148991923	0.002869381	0.106556675	0.119104567	0.145050209
