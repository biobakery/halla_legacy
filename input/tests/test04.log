2012-10-13 12:54:10,981       INFO dataset.permute@90 Generating 41 permutations
2012-10-13 12:54:10,983       INFO dataset.permute@94 Permutation 0/41
2012-10-13 12:54:10,986       INFO dataset.permute@94 Permutation 1/41
2012-10-13 12:54:10,989       INFO dataset.permute@94 Permutation 2/41
2012-10-13 12:54:10,992       INFO dataset.permute@94 Permutation 3/41
2012-10-13 12:54:10,995       INFO dataset.permute@94 Permutation 4/41
2012-10-13 12:54:10,998       INFO dataset.permute@94 Permutation 5/41
2012-10-13 12:54:11,001       INFO dataset.permute@94 Permutation 6/41
2012-10-13 12:54:11,004       INFO dataset.permute@94 Permutation 7/41
2012-10-13 12:54:11,006       INFO dataset.permute@94 Permutation 8/41
2012-10-13 12:54:11,009       INFO dataset.permute@94 Permutation 9/41
2012-10-13 12:54:11,012       INFO dataset.permute@94 Permutation 10/41
2012-10-13 12:54:11,015       INFO dataset.permute@94 Permutation 11/41
2012-10-13 12:54:11,018       INFO dataset.permute@94 Permutation 12/41
2012-10-13 12:54:11,021       INFO dataset.permute@94 Permutation 13/41
2012-10-13 12:54:11,024       INFO dataset.permute@94 Permutation 14/41
2012-10-13 12:54:11,027       INFO dataset.permute@94 Permutation 15/41
2012-10-13 12:54:11,030       INFO dataset.permute@94 Permutation 16/41
2012-10-13 12:54:11,033       INFO dataset.permute@94 Permutation 17/41
2012-10-13 12:54:11,036       INFO dataset.permute@94 Permutation 18/41
2012-10-13 12:54:11,039       INFO dataset.permute@94 Permutation 19/41
2012-10-13 12:54:11,042       INFO dataset.permute@94 Permutation 20/41
2012-10-13 12:54:11,045       INFO dataset.permute@94 Permutation 21/41
2012-10-13 12:54:11,048       INFO dataset.permute@94 Permutation 22/41
2012-10-13 12:54:11,051       INFO dataset.permute@94 Permutation 23/41
2012-10-13 12:54:11,054       INFO dataset.permute@94 Permutation 24/41
2012-10-13 12:54:11,057       INFO dataset.permute@94 Permutation 25/41
2012-10-13 12:54:11,060       INFO dataset.permute@94 Permutation 26/41
2012-10-13 12:54:11,063       INFO dataset.permute@94 Permutation 27/41
2012-10-13 12:54:11,065       INFO dataset.permute@94 Permutation 28/41
2012-10-13 12:54:11,068       INFO dataset.permute@94 Permutation 29/41
2012-10-13 12:54:11,071       INFO dataset.permute@94 Permutation 30/41
2012-10-13 12:54:11,074       INFO dataset.permute@94 Permutation 31/41
2012-10-13 12:54:11,077       INFO dataset.permute@94 Permutation 32/41
2012-10-13 12:54:11,079       INFO dataset.permute@94 Permutation 33/41
2012-10-13 12:54:11,082       INFO dataset.permute@94 Permutation 34/41
2012-10-13 12:54:11,085       INFO dataset.permute@94 Permutation 35/41
2012-10-13 12:54:11,088       INFO dataset.permute@94 Permutation 36/41
2012-10-13 12:54:11,091       INFO dataset.permute@94 Permutation 37/41
2012-10-13 12:54:11,093       INFO dataset.permute@94 Permutation 38/41
2012-10-13 12:54:11,096       INFO dataset.permute@94 Permutation 39/41
2012-10-13 12:54:11,099       INFO dataset.permute@94 Permutation 40/41
2012-10-13 12:54:11,103       INFO dataset.hierarchy@205 Cluster 1 (2): ['M1', 'D1']
2012-10-13 12:54:11,103       INFO dataset.hierarchy@205 Cluster 2 (2): ['M2', 'M3']
2012-10-13 12:54:11,103       INFO dataset.hierarchy@205 Cluster 5 (2): ['D4', 'D6']
2012-10-13 12:54:11,103       INFO dataset.bootstrap@104 Generating 100 bootstraps
2012-10-13 12:54:11,103       INFO dataset.bootstrap@107 Bootstrap 0/100
2012-10-13 12:54:11,106       INFO dataset.bootstrap@107 Bootstrap 1/100
2012-10-13 12:54:11,109       INFO dataset.bootstrap@107 Bootstrap 2/100
2012-10-13 12:54:11,112       INFO dataset.bootstrap@107 Bootstrap 3/100
2012-10-13 12:54:11,116       INFO dataset.bootstrap@107 Bootstrap 4/100
2012-10-13 12:54:11,119       INFO dataset.bootstrap@107 Bootstrap 5/100
2012-10-13 12:54:11,122       INFO dataset.bootstrap@107 Bootstrap 6/100
2012-10-13 12:54:11,125       INFO dataset.bootstrap@107 Bootstrap 7/100
2012-10-13 12:54:11,128       INFO dataset.bootstrap@107 Bootstrap 8/100
2012-10-13 12:54:11,131       INFO dataset.bootstrap@107 Bootstrap 9/100
2012-10-13 12:54:11,134       INFO dataset.bootstrap@107 Bootstrap 10/100
2012-10-13 12:54:11,137       INFO dataset.bootstrap@107 Bootstrap 11/100
2012-10-13 12:54:11,140       INFO dataset.bootstrap@107 Bootstrap 12/100
2012-10-13 12:54:11,143       INFO dataset.bootstrap@107 Bootstrap 13/100
2012-10-13 12:54:11,146       INFO dataset.bootstrap@107 Bootstrap 14/100
2012-10-13 12:54:11,149       INFO dataset.bootstrap@107 Bootstrap 15/100
2012-10-13 12:54:11,153       INFO dataset.bootstrap@107 Bootstrap 16/100
2012-10-13 12:54:11,156       INFO dataset.bootstrap@107 Bootstrap 17/100
2012-10-13 12:54:11,159       INFO dataset.bootstrap@107 Bootstrap 18/100
2012-10-13 12:54:11,161       INFO dataset.bootstrap@107 Bootstrap 19/100
2012-10-13 12:54:11,164       INFO dataset.bootstrap@107 Bootstrap 20/100
2012-10-13 12:54:11,167       INFO dataset.bootstrap@107 Bootstrap 21/100
2012-10-13 12:54:11,171       INFO dataset.bootstrap@107 Bootstrap 22/100
2012-10-13 12:54:11,174       INFO dataset.bootstrap@107 Bootstrap 23/100
2012-10-13 12:54:11,177       INFO dataset.bootstrap@107 Bootstrap 24/100
2012-10-13 12:54:11,180       INFO dataset.bootstrap@107 Bootstrap 25/100
2012-10-13 12:54:11,183       INFO dataset.bootstrap@107 Bootstrap 26/100
2012-10-13 12:54:11,186       INFO dataset.bootstrap@107 Bootstrap 27/100
2012-10-13 12:54:11,189       INFO dataset.bootstrap@107 Bootstrap 28/100
2012-10-13 12:54:11,192       INFO dataset.bootstrap@107 Bootstrap 29/100
2012-10-13 12:54:11,195       INFO dataset.bootstrap@107 Bootstrap 30/100
2012-10-13 12:54:11,198       INFO dataset.bootstrap@107 Bootstrap 31/100
2012-10-13 12:54:11,201       INFO dataset.bootstrap@107 Bootstrap 32/100
2012-10-13 12:54:11,205       INFO dataset.bootstrap@107 Bootstrap 33/100
2012-10-13 12:54:11,208       INFO dataset.bootstrap@107 Bootstrap 34/100
2012-10-13 12:54:11,211       INFO dataset.bootstrap@107 Bootstrap 35/100
2012-10-13 12:54:11,214       INFO dataset.bootstrap@107 Bootstrap 36/100
2012-10-13 12:54:11,217       INFO dataset.bootstrap@107 Bootstrap 37/100
2012-10-13 12:54:11,220       INFO dataset.bootstrap@107 Bootstrap 38/100
2012-10-13 12:54:11,223       INFO dataset.bootstrap@107 Bootstrap 39/100
2012-10-13 12:54:11,226       INFO dataset.bootstrap@107 Bootstrap 40/100
2012-10-13 12:54:11,229       INFO dataset.bootstrap@107 Bootstrap 41/100
2012-10-13 12:54:11,232       INFO dataset.bootstrap@107 Bootstrap 42/100
2012-10-13 12:54:11,236       INFO dataset.bootstrap@107 Bootstrap 43/100
2012-10-13 12:54:11,239       INFO dataset.bootstrap@107 Bootstrap 44/100
2012-10-13 12:54:11,242       INFO dataset.bootstrap@107 Bootstrap 45/100
2012-10-13 12:54:11,245       INFO dataset.bootstrap@107 Bootstrap 46/100
2012-10-13 12:54:11,248       INFO dataset.bootstrap@107 Bootstrap 47/100
2012-10-13 12:54:11,251       INFO dataset.bootstrap@107 Bootstrap 48/100
2012-10-13 12:54:11,255       INFO dataset.bootstrap@107 Bootstrap 49/100
2012-10-13 12:54:11,258       INFO dataset.bootstrap@107 Bootstrap 50/100
2012-10-13 12:54:11,261       INFO dataset.bootstrap@107 Bootstrap 51/100
2012-10-13 12:54:11,264       INFO dataset.bootstrap@107 Bootstrap 52/100
2012-10-13 12:54:11,267       INFO dataset.bootstrap@107 Bootstrap 53/100
2012-10-13 12:54:11,270       INFO dataset.bootstrap@107 Bootstrap 54/100
2012-10-13 12:54:11,273       INFO dataset.bootstrap@107 Bootstrap 55/100
2012-10-13 12:54:11,276       INFO dataset.bootstrap@107 Bootstrap 56/100
2012-10-13 12:54:11,279       INFO dataset.bootstrap@107 Bootstrap 57/100
2012-10-13 12:54:11,283       INFO dataset.bootstrap@107 Bootstrap 58/100
2012-10-13 12:54:11,286       INFO dataset.bootstrap@107 Bootstrap 59/100
2012-10-13 12:54:11,289       INFO dataset.bootstrap@107 Bootstrap 60/100
2012-10-13 12:54:11,292       INFO dataset.bootstrap@107 Bootstrap 61/100
2012-10-13 12:54:11,295       INFO dataset.bootstrap@107 Bootstrap 62/100
2012-10-13 12:54:11,298       INFO dataset.bootstrap@107 Bootstrap 63/100
2012-10-13 12:54:11,301       INFO dataset.bootstrap@107 Bootstrap 64/100
2012-10-13 12:54:11,304       INFO dataset.bootstrap@107 Bootstrap 65/100
2012-10-13 12:54:11,308       INFO dataset.bootstrap@107 Bootstrap 66/100
2012-10-13 12:54:11,311       INFO dataset.bootstrap@107 Bootstrap 67/100
2012-10-13 12:54:11,314       INFO dataset.bootstrap@107 Bootstrap 68/100
2012-10-13 12:54:11,317       INFO dataset.bootstrap@107 Bootstrap 69/100
2012-10-13 12:54:11,320       INFO dataset.bootstrap@107 Bootstrap 70/100
2012-10-13 12:54:11,323       INFO dataset.bootstrap@107 Bootstrap 71/100
2012-10-13 12:54:11,326       INFO dataset.bootstrap@107 Bootstrap 72/100
2012-10-13 12:54:11,329       INFO dataset.bootstrap@107 Bootstrap 73/100
2012-10-13 12:54:11,332       INFO dataset.bootstrap@107 Bootstrap 74/100
2012-10-13 12:54:11,335       INFO dataset.bootstrap@107 Bootstrap 75/100
2012-10-13 12:54:11,339       INFO dataset.bootstrap@107 Bootstrap 76/100
2012-10-13 12:54:11,342       INFO dataset.bootstrap@107 Bootstrap 77/100
2012-10-13 12:54:11,345       INFO dataset.bootstrap@107 Bootstrap 78/100
2012-10-13 12:54:11,348       INFO dataset.bootstrap@107 Bootstrap 79/100
2012-10-13 12:54:11,351       INFO dataset.bootstrap@107 Bootstrap 80/100
2012-10-13 12:54:11,354       INFO dataset.bootstrap@107 Bootstrap 81/100
2012-10-13 12:54:11,357       INFO dataset.bootstrap@107 Bootstrap 82/100
2012-10-13 12:54:11,360       INFO dataset.bootstrap@107 Bootstrap 83/100
2012-10-13 12:54:11,363       INFO dataset.bootstrap@107 Bootstrap 84/100
2012-10-13 12:54:11,366       INFO dataset.bootstrap@107 Bootstrap 85/100
2012-10-13 12:54:11,369       INFO dataset.bootstrap@107 Bootstrap 86/100
2012-10-13 12:54:11,372       INFO dataset.bootstrap@107 Bootstrap 87/100
2012-10-13 12:54:11,375       INFO dataset.bootstrap@107 Bootstrap 88/100
2012-10-13 12:54:11,378       INFO dataset.bootstrap@107 Bootstrap 89/100
2012-10-13 12:54:11,381       INFO dataset.bootstrap@107 Bootstrap 90/100
2012-10-13 12:54:11,385       INFO dataset.bootstrap@107 Bootstrap 91/100
2012-10-13 12:54:11,388       INFO dataset.bootstrap@107 Bootstrap 92/100
2012-10-13 12:54:11,391       INFO dataset.bootstrap@107 Bootstrap 93/100
2012-10-13 12:54:11,394       INFO dataset.bootstrap@107 Bootstrap 94/100
2012-10-13 12:54:11,397       INFO dataset.bootstrap@107 Bootstrap 95/100
2012-10-13 12:54:11,400       INFO dataset.bootstrap@107 Bootstrap 96/100
2012-10-13 12:54:11,403       INFO dataset.bootstrap@107 Bootstrap 97/100
2012-10-13 12:54:11,406       INFO dataset.bootstrap@107 Bootstrap 98/100
2012-10-13 12:54:11,409       INFO dataset.bootstrap@107 Bootstrap 99/100
2012-10-13 12:54:11,414       INFO halla._halla_test@82 Testing 0/8
2012-10-13 12:54:11,423       INFO halla._halla_test@82 Testing 1/8
2012-10-13 12:54:11,431       INFO halla._halla_test@82 Testing 2/8
2012-10-13 12:54:11,439       INFO halla._halla_test@82 Testing 3/8
2012-10-13 12:54:11,444       INFO halla._halla_test@82 Testing 4/8
2012-10-13 12:54:11,448       INFO halla._halla_test@82 Testing 5/8
2012-10-13 12:54:11,451       INFO halla._halla_test@82 Testing 6/8
2012-10-13 12:54:11,453       INFO halla._halla_test@82 Testing 7/8
