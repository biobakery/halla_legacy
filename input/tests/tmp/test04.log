2012-10-13 11:34:55,746       INFO dataset.permute@90 Generating 41 permutations
2012-10-13 11:34:55,749       INFO dataset.permute@94 Permutation 0/41
2012-10-13 11:34:55,755       INFO dataset.permute@94 Permutation 1/41
2012-10-13 11:34:55,760       INFO dataset.permute@94 Permutation 2/41
2012-10-13 11:34:55,766       INFO dataset.permute@94 Permutation 3/41
2012-10-13 11:34:55,772       INFO dataset.permute@94 Permutation 4/41
2012-10-13 11:34:55,777       INFO dataset.permute@94 Permutation 5/41
2012-10-13 11:34:55,783       INFO dataset.permute@94 Permutation 6/41
2012-10-13 11:34:55,788       INFO dataset.permute@94 Permutation 7/41
2012-10-13 11:34:55,794       INFO dataset.permute@94 Permutation 8/41
2012-10-13 11:34:55,799       INFO dataset.permute@94 Permutation 9/41
2012-10-13 11:34:55,804       INFO dataset.permute@94 Permutation 10/41
2012-10-13 11:34:55,809       INFO dataset.permute@94 Permutation 11/41
2012-10-13 11:34:55,814       INFO dataset.permute@94 Permutation 12/41
2012-10-13 11:34:55,819       INFO dataset.permute@94 Permutation 13/41
2012-10-13 11:34:55,824       INFO dataset.permute@94 Permutation 14/41
2012-10-13 11:34:55,830       INFO dataset.permute@94 Permutation 15/41
2012-10-13 11:34:55,836       INFO dataset.permute@94 Permutation 16/41
2012-10-13 11:34:55,841       INFO dataset.permute@94 Permutation 17/41
2012-10-13 11:34:55,847       INFO dataset.permute@94 Permutation 18/41
2012-10-13 11:34:55,852       INFO dataset.permute@94 Permutation 19/41
2012-10-13 11:34:55,855       INFO dataset.permute@94 Permutation 20/41
2012-10-13 11:34:55,858       INFO dataset.permute@94 Permutation 21/41
2012-10-13 11:34:55,861       INFO dataset.permute@94 Permutation 22/41
2012-10-13 11:34:55,864       INFO dataset.permute@94 Permutation 23/41
2012-10-13 11:34:55,867       INFO dataset.permute@94 Permutation 24/41
2012-10-13 11:34:55,870       INFO dataset.permute@94 Permutation 25/41
2012-10-13 11:34:55,873       INFO dataset.permute@94 Permutation 26/41
2012-10-13 11:34:55,876       INFO dataset.permute@94 Permutation 27/41
2012-10-13 11:34:55,879       INFO dataset.permute@94 Permutation 28/41
2012-10-13 11:34:55,882       INFO dataset.permute@94 Permutation 29/41
2012-10-13 11:34:55,885       INFO dataset.permute@94 Permutation 30/41
2012-10-13 11:34:55,888       INFO dataset.permute@94 Permutation 31/41
2012-10-13 11:34:55,891       INFO dataset.permute@94 Permutation 32/41
2012-10-13 11:34:55,894       INFO dataset.permute@94 Permutation 33/41
2012-10-13 11:34:55,897       INFO dataset.permute@94 Permutation 34/41
2012-10-13 11:34:55,900       INFO dataset.permute@94 Permutation 35/41
2012-10-13 11:34:55,903       INFO dataset.permute@94 Permutation 36/41
2012-10-13 11:34:55,906       INFO dataset.permute@94 Permutation 37/41
2012-10-13 11:34:55,909       INFO dataset.permute@94 Permutation 38/41
2012-10-13 11:34:55,912       INFO dataset.permute@94 Permutation 39/41
2012-10-13 11:34:55,915       INFO dataset.permute@94 Permutation 40/41
2012-10-13 11:34:55,919       INFO dataset.hierarchy@205 Cluster 1 (2): ['M1', 'D1']
2012-10-13 11:34:55,919       INFO dataset.hierarchy@205 Cluster 2 (2): ['M2', 'M3']
2012-10-13 11:34:55,919       INFO dataset.hierarchy@205 Cluster 5 (2): ['D4', 'D6']
2012-10-13 11:34:55,920       INFO dataset.bootstrap@104 Generating 100 bootstraps
2012-10-13 11:34:55,920       INFO dataset.bootstrap@107 Bootstrap 0/100
2012-10-13 11:34:55,923       INFO dataset.bootstrap@107 Bootstrap 1/100
2012-10-13 11:34:55,926       INFO dataset.bootstrap@107 Bootstrap 2/100
2012-10-13 11:34:55,930       INFO dataset.bootstrap@107 Bootstrap 3/100
2012-10-13 11:34:55,933       INFO dataset.bootstrap@107 Bootstrap 4/100
2012-10-13 11:34:55,937       INFO dataset.bootstrap@107 Bootstrap 5/100
2012-10-13 11:34:55,940       INFO dataset.bootstrap@107 Bootstrap 6/100
2012-10-13 11:34:55,943       INFO dataset.bootstrap@107 Bootstrap 7/100
2012-10-13 11:34:55,946       INFO dataset.bootstrap@107 Bootstrap 8/100
2012-10-13 11:34:55,950       INFO dataset.bootstrap@107 Bootstrap 9/100
2012-10-13 11:34:55,953       INFO dataset.bootstrap@107 Bootstrap 10/100
2012-10-13 11:34:55,956       INFO dataset.bootstrap@107 Bootstrap 11/100
2012-10-13 11:34:55,959       INFO dataset.bootstrap@107 Bootstrap 12/100
2012-10-13 11:34:55,963       INFO dataset.bootstrap@107 Bootstrap 13/100
2012-10-13 11:34:55,966       INFO dataset.bootstrap@107 Bootstrap 14/100
2012-10-13 11:34:55,969       INFO dataset.bootstrap@107 Bootstrap 15/100
2012-10-13 11:34:55,973       INFO dataset.bootstrap@107 Bootstrap 16/100
2012-10-13 11:34:55,976       INFO dataset.bootstrap@107 Bootstrap 17/100
2012-10-13 11:34:55,980       INFO dataset.bootstrap@107 Bootstrap 18/100
2012-10-13 11:34:55,983       INFO dataset.bootstrap@107 Bootstrap 19/100
2012-10-13 11:34:55,986       INFO dataset.bootstrap@107 Bootstrap 20/100
2012-10-13 11:34:55,990       INFO dataset.bootstrap@107 Bootstrap 21/100
2012-10-13 11:34:55,993       INFO dataset.bootstrap@107 Bootstrap 22/100
2012-10-13 11:34:55,996       INFO dataset.bootstrap@107 Bootstrap 23/100
2012-10-13 11:34:56,000       INFO dataset.bootstrap@107 Bootstrap 24/100
2012-10-13 11:34:56,003       INFO dataset.bootstrap@107 Bootstrap 25/100
2012-10-13 11:34:56,007       INFO dataset.bootstrap@107 Bootstrap 26/100
2012-10-13 11:34:56,010       INFO dataset.bootstrap@107 Bootstrap 27/100
2012-10-13 11:34:56,013       INFO dataset.bootstrap@107 Bootstrap 28/100
2012-10-13 11:34:56,017       INFO dataset.bootstrap@107 Bootstrap 29/100
2012-10-13 11:34:56,021       INFO dataset.bootstrap@107 Bootstrap 30/100
2012-10-13 11:34:56,024       INFO dataset.bootstrap@107 Bootstrap 31/100
2012-10-13 11:34:56,027       INFO dataset.bootstrap@107 Bootstrap 32/100
2012-10-13 11:34:56,031       INFO dataset.bootstrap@107 Bootstrap 33/100
2012-10-13 11:34:56,034       INFO dataset.bootstrap@107 Bootstrap 34/100
2012-10-13 11:34:56,037       INFO dataset.bootstrap@107 Bootstrap 35/100
2012-10-13 11:34:56,040       INFO dataset.bootstrap@107 Bootstrap 36/100
2012-10-13 11:34:56,044       INFO dataset.bootstrap@107 Bootstrap 37/100
2012-10-13 11:34:56,048       INFO dataset.bootstrap@107 Bootstrap 38/100
2012-10-13 11:34:56,051       INFO dataset.bootstrap@107 Bootstrap 39/100
2012-10-13 11:34:56,054       INFO dataset.bootstrap@107 Bootstrap 40/100
2012-10-13 11:34:56,058       INFO dataset.bootstrap@107 Bootstrap 41/100
2012-10-13 11:34:56,061       INFO dataset.bootstrap@107 Bootstrap 42/100
2012-10-13 11:34:56,065       INFO dataset.bootstrap@107 Bootstrap 43/100
2012-10-13 11:34:56,068       INFO dataset.bootstrap@107 Bootstrap 44/100
2012-10-13 11:34:56,071       INFO dataset.bootstrap@107 Bootstrap 45/100
2012-10-13 11:34:56,075       INFO dataset.bootstrap@107 Bootstrap 46/100
2012-10-13 11:34:56,078       INFO dataset.bootstrap@107 Bootstrap 47/100
2012-10-13 11:34:56,082       INFO dataset.bootstrap@107 Bootstrap 48/100
2012-10-13 11:34:56,085       INFO dataset.bootstrap@107 Bootstrap 49/100
2012-10-13 11:34:56,089       INFO dataset.bootstrap@107 Bootstrap 50/100
2012-10-13 11:34:56,092       INFO dataset.bootstrap@107 Bootstrap 51/100
2012-10-13 11:34:56,096       INFO dataset.bootstrap@107 Bootstrap 52/100
2012-10-13 11:34:56,099       INFO dataset.bootstrap@107 Bootstrap 53/100
2012-10-13 11:34:56,102       INFO dataset.bootstrap@107 Bootstrap 54/100
2012-10-13 11:34:56,106       INFO dataset.bootstrap@107 Bootstrap 55/100
2012-10-13 11:34:56,109       INFO dataset.bootstrap@107 Bootstrap 56/100
2012-10-13 11:34:56,113       INFO dataset.bootstrap@107 Bootstrap 57/100
2012-10-13 11:34:56,116       INFO dataset.bootstrap@107 Bootstrap 58/100
2012-10-13 11:34:56,119       INFO dataset.bootstrap@107 Bootstrap 59/100
2012-10-13 11:34:56,123       INFO dataset.bootstrap@107 Bootstrap 60/100
2012-10-13 11:34:56,126       INFO dataset.bootstrap@107 Bootstrap 61/100
2012-10-13 11:34:56,130       INFO dataset.bootstrap@107 Bootstrap 62/100
2012-10-13 11:34:56,133       INFO dataset.bootstrap@107 Bootstrap 63/100
2012-10-13 11:34:56,137       INFO dataset.bootstrap@107 Bootstrap 64/100
2012-10-13 11:34:56,140       INFO dataset.bootstrap@107 Bootstrap 65/100
2012-10-13 11:34:56,144       INFO dataset.bootstrap@107 Bootstrap 66/100
2012-10-13 11:34:56,147       INFO dataset.bootstrap@107 Bootstrap 67/100
2012-10-13 11:34:56,151       INFO dataset.bootstrap@107 Bootstrap 68/100
2012-10-13 11:34:56,154       INFO dataset.bootstrap@107 Bootstrap 69/100
2012-10-13 11:34:56,158       INFO dataset.bootstrap@107 Bootstrap 70/100
2012-10-13 11:34:56,161       INFO dataset.bootstrap@107 Bootstrap 71/100
2012-10-13 11:34:56,165       INFO dataset.bootstrap@107 Bootstrap 72/100
2012-10-13 11:34:56,168       INFO dataset.bootstrap@107 Bootstrap 73/100
2012-10-13 11:34:56,172       INFO dataset.bootstrap@107 Bootstrap 74/100
2012-10-13 11:34:56,175       INFO dataset.bootstrap@107 Bootstrap 75/100
2012-10-13 11:34:56,178       INFO dataset.bootstrap@107 Bootstrap 76/100
2012-10-13 11:34:56,182       INFO dataset.bootstrap@107 Bootstrap 77/100
2012-10-13 11:34:56,185       INFO dataset.bootstrap@107 Bootstrap 78/100
2012-10-13 11:34:56,189       INFO dataset.bootstrap@107 Bootstrap 79/100
2012-10-13 11:34:56,192       INFO dataset.bootstrap@107 Bootstrap 80/100
2012-10-13 11:34:56,195       INFO dataset.bootstrap@107 Bootstrap 81/100
2012-10-13 11:34:56,199       INFO dataset.bootstrap@107 Bootstrap 82/100
2012-10-13 11:34:56,202       INFO dataset.bootstrap@107 Bootstrap 83/100
2012-10-13 11:34:56,206       INFO dataset.bootstrap@107 Bootstrap 84/100
2012-10-13 11:34:56,209       INFO dataset.bootstrap@107 Bootstrap 85/100
2012-10-13 11:34:56,212       INFO dataset.bootstrap@107 Bootstrap 86/100
2012-10-13 11:34:56,216       INFO dataset.bootstrap@107 Bootstrap 87/100
2012-10-13 11:34:56,220       INFO dataset.bootstrap@107 Bootstrap 88/100
2012-10-13 11:34:56,223       INFO dataset.bootstrap@107 Bootstrap 89/100
2012-10-13 11:34:56,227       INFO dataset.bootstrap@107 Bootstrap 90/100
2012-10-13 11:34:56,230       INFO dataset.bootstrap@107 Bootstrap 91/100
2012-10-13 11:34:56,234       INFO dataset.bootstrap@107 Bootstrap 92/100
2012-10-13 11:34:56,240       INFO dataset.bootstrap@107 Bootstrap 93/100
2012-10-13 11:34:56,246       INFO dataset.bootstrap@107 Bootstrap 94/100
2012-10-13 11:34:56,253       INFO dataset.bootstrap@107 Bootstrap 95/100
2012-10-13 11:34:56,260       INFO dataset.bootstrap@107 Bootstrap 96/100
2012-10-13 11:34:56,267       INFO dataset.bootstrap@107 Bootstrap 97/100
2012-10-13 11:34:56,271       INFO dataset.bootstrap@107 Bootstrap 98/100
2012-10-13 11:34:56,276       INFO dataset.bootstrap@107 Bootstrap 99/100
2012-10-13 11:34:56,284       INFO halla._halla_test@82 Testing 0/8
2012-10-13 11:34:56,302       INFO halla._halla_test@82 Testing 1/8
2012-10-13 11:34:56,317       INFO halla._halla_test@82 Testing 2/8
2012-10-13 11:34:56,330       INFO halla._halla_test@82 Testing 3/8
2012-10-13 11:34:56,340       INFO halla._halla_test@82 Testing 4/8
2012-10-13 11:34:56,345       INFO halla._halla_test@82 Testing 5/8
2012-10-13 11:34:56,349       INFO halla._halla_test@82 Testing 6/8
2012-10-13 11:34:56,350       INFO halla._halla_test@82 Testing 7/8
