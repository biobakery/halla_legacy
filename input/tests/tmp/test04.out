Clusters
M1	D1
M2	M3
D4	D6
One	Two	MI	MID	Pperm	Pboot	P
M1	M2	0.666667	0.333333	0.0238095	4.43404e-11	0.0238095
M1	D3	0.699197	0.300803	0.0238095	6.21129e-28	0.0238095
M1	D2	0.647317	0.352683	0.0238095	1.24855e-19	0.0238095
M2	D3	0.978118	0.382876	0.0238095	8.33745e-13	0.0238095
M2	D2	1.08188	0.317411	0.0238095	2.43015e-21	0.0238095
D3	D2	0.97611	0.579612	0.047619	3.19948e-08	0.047619
D3	D4	0.579869	0.750264	0.595238	0.00562748	0.595238
D3	D7	0.663203	0.714374	0.428571	0.00661718	0.428571
D3	D5	0.694656	0.700828	0.309524	0.000194384	0.309524
D2	D4	0.631749	0.727921	0.547619	0.00169985	0.547619
D2	D7	0.465082	0.7997	0.821429	0.028995	0.821429
D2	D5	0.465082	0.7997	0.869048	0.00794631	0.869048
D4	D7	0.548416	0.76381	0.821429	0.0100804	0.821429
D4	D5	0.663203	0.714374	0.559524	9.20288e-05	0.559524
D7	D5	0.496536	0.786154	0.857143	0.0249469	0.857143
